sdk/core/azure-core/src/http/policy.cpp

```diff
@@ -0,0 +1,21 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// SPDX-License-Identifier: MIT
+
+#include <http/http.hpp>
+#include <http/policy.hpp>
+
+using namespace Azure::Core::Http;
+
+std::unique_ptr<Response> NextHttpPolicy::ProcessNext(Context& ctx, Request& req)
+{
+  if (m_policies == nullptr)
+    throw;
+
+  if (m_index == m_policies->size() - 1)
+  {
+    //All the policies have run without running a transport policy
+    throw;
```

Comment:
Message: Your comment can be the message

Reply:
Again, I'll make the errors better once the architecture is agreed upon